### ISSUE TITLE
Prefix get added on the wrong devices

### ISSUE THREAD

**Issue by IanCStewart** [FormidableLabs/radium#963]
When passing `display: flex` to Radium it prefixes `flex` to `-webkit-box` for samsung browsers. When looking at the specs of this browser it accepts `display: flex`. As seen on the images below one component with and one component with out export `Radium(component)`. The one without renders as expected.

<img width="889" alt="screen shot 2018-01-22 at 14 17 58" src="https://user-images.githubusercontent.com/14125280/35222802-6207e994-ff7f-11e7-99fb-003558fe3004.png">

_With Radium_

<img width="669" alt="screen shot 2018-01-22 at 14 18 15" src="https://user-images.githubusercontent.com/14125280/35222803-621ffd7c-ff7f-11e7-9143-50f2a84f11dd.png">

_Without Radium_

_To recap: Radium prefixes wrong for samsung browser when using flex_

**Comment by ryan-roemer** [comment]
@IanCStewart -- Which version of Radium are you on?

Also, what's the user agent for the samsung browser you're using? As far as I understand, we just use https://github.com/rofrischmann/inline-style-prefixer so perhaps you can also investigate if there is a bug upstream with them?

**Comment by IanCStewart** [comment]
I was using version `^0.19.4` this seemed to be the problem when updated to the latest version it seems to work fine.

In case you're still wondering about my user agent: `Mozilla/5.0 (Linux; Android 6.0.1; SAMSUNG SM-G900F Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) SamsungBrowser/4.0 Chrome/44.0.2403.133 Mobile Safari/537.36`.

I'll close this issue thanks for your help!
